src/backend/distributed/executor/multi_client_executor.c

```diff
@@ -854,23 +857,6 @@ MultiClientWait(WaitInfo *waitInfo)
 
 
 /*
- * ClearRemainingResults reads result objects from the connection until we get
- * null, and clears these results. This is the last step in completing an async
- * query.
- */
-static void
-ClearRemainingResults(MultiConnection *connection)
```

Comment:
In the deadlock meeting Ozgun brought up that `ForgetResults` is a strange name.